src/content/get-started/using-okteto-cli-and-dashboard.mdx

```diff
@@ -0,0 +1,109 @@
+
+A list of all available CLI commands is in our full [CLI reference guide here ->](reference/okteto-cli.mdx)
+
+## Overview of the Okteto Dashboard
+
+### 1. **Your Personal Namespace**
+
+  Automatically generated for each user, this Personal Namespace is uniquely named after your username or email, ensuring a personalized environment. These cannot be deleted, but their resources can be destroyed allowing the Personal Namespace to be reused.
+
+   <p align="center">
+      <Image
+         src={require('@site/static/img/namespace-landing.jpg').default}
+         alt="Okteto Namespace Landing Page"
+         width="800"
+      />
+   </p>
+
+### 2. **Using the Catalog to Deploy a Development Environment**
+
+   The most common way to deploy a Development Environment on Okteto is through the Catalog. The Catalog is a list of your applications, configured by a platform engineer on your team, with everything the application needs to run included. You can find the Catalog by clicking on the “Deploy a Dev Environment” button.
+
+   <p align="center">
+      <Image
+         src={require('@site/static/img/catalog-deployment.jpg').default}
+         alt="Okteto Namespace Landing Page"
+         width="800"
+      />
+   </p>
+
+   You can also deploy a Development Environment with the following methods:
+
+   - **Git Repository:** Deploy from a repository in a connected GitHub account
+   - **URL**: Deploy from a specific Git URL
+
+### 3. **Creating a New Namespace**
+   - **Non-Personal Namespaces** are additional Namespaces created by users or through automation, facilitating broader project collaboration and management. They may be deleted or transferred to other users.
+   - As a common practice, you might name your Namespace based on a feature number, ticket number, or a similar identifier. Development environments within these Namespaces are created from Git branches. This allows you to run multiple Namespaces simultaneously, each corresponding to a different branch or feature.
+
+### 4. [**Sharing a Namespace**](https://www.okteto.com/docs/core/namespaces/#sharing-and-collaboration-with-namespaces)
+
+   - Collaborate on several applications at once by sharing your namespace
+   - To share a Namespace go to the Okteto dashboard, select the namespace you want to share and press the `Share` button in the namespace menu (you'll find it in the main bar at the top).
+
+   <p align="center">
+      <Image
+         src={require('@site/static/img/sharing-a-namespace.jpg').default}
+         alt="Okteto Namespace Landing Page"
+         width="800"
+      />
+   </p>
+
+### 5. **Environment States**
+   - **Sleep**: Suspend your environment to save resources, the Okteto Garbage collector configured by your Okteto admin will set the sleep periods after a certain amount of inactivity
+   - **Destroy All**: Remove all resources in a Namespace
+   - **Keep Awake**: Admins can choose to prevent Namespaces from sleeping so that they are always accessible
```

Comment:
If this is for admins, I wouldn't mention it in a getting started for developers guide